src/interface/web/app/settings/page.tsx

```diff
@@ -0,0 +1,999 @@
+            <div className="flex flex-col h-full">
+                <div className="flex-none p-4 bg-background border-b">
+                    <CommandInput
+                        placeholder="Find synced files"
+                        value={searchQuery}
+                        onValueChange={setSearchQuery}
+                    />
+                </div>
+
+                <div className="flex-grow overflow-auto">
+                    <CommandList>
+                        <CommandEmpty>No such files synced.</CommandEmpty>
+                        <CommandGroup heading="Synced files">
+                            {filteredFiles.map((filename: string) => (
+                                <CommandItem
+                                    key={filename}
+                                    value={filename}
+                                    onSelect={(value) => {
+                                        setSelectedFiles(prev =>
+                                            prev.includes(value)
+                                                ? prev.filter(f => f !== value)
+                                                : [...prev, value]
+                                        );
+                                    }}
+                                >
+                                    <div className="flex items-center justify-between w-full">
+                                        <div className={`flex items-center ${selectedFiles.includes(filename) ? 'font-semibold' : ''}`}>
+                                            {selectedFiles.includes(filename) && <Check className="h-4 w-4 mr-2" />}
+                                            <span className="break-all">{filename}</span>
+                                        </div>
+                                        <Button
+                                            variant="outline"
+                                            size="sm"
+                                            onClick={() => deleteFile(filename)}
+                                            className="ml-auto"
+                                        >
+                                            <Trash className="h-4 w-4" />
+                                        </Button>
+                                    </div>
+                                </CommandItem>
+                            ))}
+                        </CommandGroup>
+                    </CommandList>
+                </div>
+
+                <div className="flex-none p-4 bg-background border-t">
+                    <div className="flex justify-between">
+                        <Button
+                            variant="outline"
+                            size="sm"
+                            onClick={deleteSelected}
+                            className="mr-2"
+                        >
+                            <Trash className="h-4 w-4 mr-2" />
+                            Delete All
```

Comment:
Can we update this text to say `Delete ${deleteSelected.length} files` when some files are selected?